src/main/resources/db/migration/veilarbregistreringDB/V1_38__fk_fra_registreringtilstand_til_brukerregistrering.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE REGISTRERING_TILSTAND ADD CONSTRAINT "BRUKER_REGISTRERING_ID_FK" FOREIGN KEY ("BRUKER_REGISTRERING_ID") REFERENCES BRUKER_REGISTRERING ("BRUKER_REGISTRERING_ID");
```

Comment:
Jeg kunne kanskje hatt lyst til å kjørt FK_BRUKER_REGISTRERING_ID, på lik linje med PK_... for den jeg laget sist, men det er nok slik du har gjort det som har vært vanlig i prosjektet fra før. Synes det blir tydeligere hva som er FK og PK når vi har det først.

Men når jeg googler "naming convention og Oracle", så virker det som det er mest vanlig å ha det til slutt som her, så kjør på. :)

Reply:
Haha, den er god! 😄

Skulle vi da også vurderte å migrere navnet i tilstandstabellen til dette formatet?